File: Dante1/Module1.01/Zadanie1.20/main.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    float num1, num2, num3;

    float kut1;
    float kut2;

    printf("Enter num1: ");
    int check1 = scanf("%f", &num1);
    if(check1 != 1)
    {
        return 1;
    }

    printf("Enter num2: ");
    int check2 = scanf("%f", &num2);
    if(check2 != 1)
    {
        return 1;
    }

    num3 = sqrt(num1 * num1 + num2 * num2);
    printf("Result is: %.2f\n", num3);

    kut1 = asin((num1 / num3)) * 180 / 3.14159265359;
    kut2 = asin((num2 / num3)) * 180 / 3.14159265359;

    printf("%.2f\n", kut1);
    printf("%.2f", kut2);
    return 0;
}

File: Dante1/Module1.01/Zadanie1.20/main.rs
fn main()
{
    println!("Enter a: ");
    let mut a = String::new();
    std::io::stdin().read_line(&mut a).expect("Failed to read line");
    let a: f64 = a.trim().parse().expect("Invalid input");

    println!("Enter b: ");
    let mut b = String::new();
    std::io::stdin().read_line(&mut b).expect("Failed to read line");
    let b: f64 = b.trim().parse().expect("Invalid input");

    let c: f64 = (a.powi(2) + b.powi(2)).sqrt();
    println!("C is: {:.2}", c);

    let kut1: f64 = (a / c).asin() * 180.0 / std::f64::consts::PI;
    let kut2: f64 = (b / c).asin() * 180.0 / std::f64::consts::PI;

    println!("{:.2}", kut1);
    println!("{:.2}", kut2);
    
    return;
}
